### DISCUSSION TITLE
WHY ARE PEOPLE DOWNLOADING LOG5J?

### DISCUSSION THREAD

**Discussion by tom-ricci** [XaverianTeamRobotics/CenterstageFTCrobotcontroller#69] [Show and tell]
# THAT WASN'T SUPPOSED TO HAPPEN!

![image](https://user-images.githubusercontent.com/79022795/146195645-2b2270a4-b852-4780-acb7-0eaae0fb32aa.png)

*Edit: You are legally required to use proper grammar.*

**Comment by tom-ricci** [comment]
i also got it to log stuff and still stay 420 bytes so. yes

**Reply by MatthewL246** [reply]
Good. You are singlehandedly improving the security of the entire Internet.

**Comment by tom-ricci** [comment]
lmao i even managed to fit a license in 420 too

**Comment by MatthewL246** [comment]
Thank you for your amazing contribution to open source! 🎉

**Reply by MatthewL246** [reply]
Oh, wait, is it actually even open source?

**Reply by tom-ricci** [reply]
yes i made a custom license thats 43 bytes long

**Reply by MatthewL246** [reply]
Ah yes, the `Pubdom Ver 1639577438` license.

**Comment by MatthewL246** [comment]
Unfortunately, it's probably mostly bots that archive packages and stuff. 😢

**Comment by tom-ricci** [comment]
OH MY GOD I THINK I JUST BROKE THE ENTIRETY OF NPM

I AM 100% SURE IT IS NOT THE WIFI

Edit: You didn't lol.
Edit 2: I love how GitHub collaborators can just edit anyone's comment.

**Reply by tom-ricci** [reply]
OKAY. LITERALLY. I UPLOAD A PACKAGE BY ACCIDENT. I CANCEL THE UPLOAD OPERATION. SOMEHOW. AND I DO NOT KNOW HOW. BUT SOMEHOW THE ENTIRETY OF NPM IS RETURNING A 503. WHAT

**Reply by tom-ricci** [reply]
wait maybe not if npm runs on aws theyre down thats probably why aw i thought i did something

**Reply by tom-ricci** [reply]
aw i didnt do anything just the entire worlds down oh well

**Comment by tom-ricci** [comment]
lol the compressed size is bigger than the uncompressed size
![image](https://user-images.githubusercontent.com/79022795/146220833-ae8d1e65-feca-4922-854a-55fbd07797b7.png)

**Comment by MatthewL246** [comment]
## **Noooo it's unpublished!**

```js
npm ERR! code E404
npm ERR! 404 Unpublished by undefined on 2021-12-15T15:58:10.580Z

npm ERR! 404
npm ERR! 404  'log5j' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git URL.
```

![WOMBAT COMBAT](https://static.npmjs.com/2f55441126e3c8d643c4c2d4e852cb4c.png)

**Reply by MatthewL246** [reply]
`Unpublished by undefined`, huh? Sounds like an NPM admin didn't like your package.

**Reply by tom-ricci** [reply]
itll be back Soon:tm:
